next permutation in c++

```diff
@@ -0,0 +1,69 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+//to print a vector
+void print(vector<int> a,int n){ 
```

Comment:
There are several comments here:
1. Naming: Google style requires `MixedCase` (see https://google.github.io/styleguide/cppguide.html#Function_Names), so this should be `Print`. A better name might be `PrintVector`, though. The parameter can also be named better: `vec`, `vector` or even `v` would improve readability here.
2. The `std::vector<int>` is needlessly copied on each invocation of this function, so you should use a const reference: `const std::vector<int>& a`. Should you later want to use a different container (like `std::array<>` or a plain array), you could also accept an `absl::Span`. See https://abseil.io/tips/93 for how to use that.
3. The `n` parameter is not necessary, as the vector already has a `size()` method, giving the number of elements currently in the vector.